java/src/main/java/com/powsybl/dataframe/network/NetworkDataframes.java

```diff
@@ -860,6 +864,50 @@ static String getPhaseTapChangerRegulatedSide(TwoWindingsTransformer transformer
         return getTerminalSideStr(transformer, transformer.getPhaseTapChanger().getRegulationTerminal());
     }
 
+    static NetworkDataframeMapper identifiables() {
+        return NetworkDataframeMapperBuilder.ofStream(network -> network.getIdentifiables().stream(), NetworkDataframes::getT2OrThrow)
+                .stringsIndex("id", Identifiable::getId)
+                .strings("type", identifiable -> identifiable.getType().toString())
+                .build();
+    }
+
+    static NetworkDataframeMapper injections() {
+        return NetworkDataframeMapperBuilder.ofStream(network -> network.getConnectableStream()
+                        .filter(connectable -> connectable instanceof Injection),
+                NetworkDataframes::getT2OrThrow)
+                .stringsIndex("id", Connectable::getId)
+                .strings("type", connectable -> connectable.getType().toString())
+                .strings("voltage_level_id", connectable -> ((Injection) connectable).getTerminal().getVoltageLevel().getId())
+                .strings("bus_id", connectable -> ((Injection) connectable).getTerminal().getBusView().getBus() == null ? "" :
+                        ((Injection) connectable).getTerminal().getBusView().getBus().getId())
+                .build();
+    }
+
+    static NetworkDataframeMapper branches() {
```

Comment:
Maybe we should add hvdc line and the legs of the 3 windings transformers ? @sylvlecl

Reply:
Not really in favor of it because in all other places in powsybl branches are only lines + 2 windings transformers (elements that connect 2 terminals). Or we need to give it a different name ?

The 3 windings transformers will not fit well since there will be 3 voltages levels.

Reply:
Do we create another dataframe with 2 windings, 3 windings legs, hvdc and lines ? @annetill @sylvlecl

Reply:
I understand for three windings transformers even if my experience is that sometimes I really miss something really simple that gives me the buses and the edges, as the simplest model for calculations. For HVDC lines, I think it is questionnable not to have it. I know that we never put them in branches in Java. But we are in pypowsybl and we have to ease network analysis.

Reply:
to conclude what do I do ? I let a dataframe with the branches of lines and 2 windings transformers ? and no other dataframes ?

Reply:
yes